docs/ref/contrib/staticfiles.txt

```diff
@@ -285,11 +285,17 @@ following requirements are met:
 * the :setting:`STATICFILES_STORAGE` setting is set to
   ``'django.contrib.staticfiles.storage.ManifestStaticFilesStorage'``
 * the :setting:`DEBUG` setting is set to ``False``
-* you use the ``staticfiles`` :ttag:`static<staticfiles-static>` template
-  tag to refer to your static files in your templates
 * you've collected all your static files by using the
   :djadmin:`collectstatic` management command
 
+.. versionchanged:: 1.10
+
+    * you no longer have to use ``staticfiles`` :ttag:`static<staticfiles-static>`
+      template tag to refer to your static files in your templates.
+      The ``static`` template tag uses the ``staticfiles`` template tag,
```

Comment:
This seems slightly confusing to me (although it's the fault of history). We're talking specifically about the `static` template tag in the `static` template library, which will now use the `static` template tag in the `staticfiles` template library (if `staticfiles` is enabled).

This might be better phrased by focussing on the impact, ie that if `staticfiles` is enabled, the `static` template *library* will automatically use it, rather than trying to describe exactly what the change is?